### ISSUE TITLE
Update OCR Add-on to use config.conf

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#6093]
One of the most referenced add-ons for new devs is the OCR plugin. we should either update it to use NVDA's core configuration, as an example of how it should be done, or we should put comments in the code explaining that this method isn't the best way to do config unless there's a good reason such as storage rather than config or another similar issue.

**Comment by jcsteh** [comment]
Already filed as nvaccess/nvda-ocr#2, which is the correct place for nvda-ocr issues. Thanks.
